thunder/__init__.py

```diff
@@ -503,6 +509,9 @@ def get_computation_and_inputs(*args, **kwargs):
 
             prologue_traces = [prologue_trc]
             computation_traces = [computation_trc]
+            if not compile_options.get("skip_inplace_functionalization", False):
+                computation_traces.extend(functionalize_inplace_ops(computation_trace=computation_trc))
+                computation_trc = computation_traces[-1]
```

Comment:
Longer term, I wonder if this should be a "default transform", but maybe it is important that this goes first and so it is tricky with the timing.